_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,349 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: composite
+
+description: |
+  This rule checks that links that are embedded in a paragraph, list or cell have a way to distinguish them from the surrounding text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_rules:
+  - e7cc2b
+  - 88407d
+  - 36f116
+  - 66e9f0
+  - 548868
+  - 2803b8
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- has [descendant][] [text nodes][text node] in the [flat tree][] that are [visible][]; and
+- is a [descendant][] in the [flat tree][] of an HTML element (with the closest [ancestor][] in the [flat tree][] being henceforth referred to as the _ancestor_ element) that has a [computed value][] for the [display property][display] that is not one of the following: 'table', 'flex', 'grid', 'ruby', 'table-row-group', 'table-header-group', 'table-footer-group', 'table-row', table-column-group', 'table-column', 'ruby-base-container' and 'ruby-text-container'; and
```

Comment:
Which atomic rule does the `display` requirement come from? I can't seem to find it 🤔

Reply:
It wasn't from a specific atomic rule, but from a general comment from @WilcoFiers in this thread: https://github.com/act-rules/act-rules.github.io/pull/1010#discussion_r390865422

Reply:
Ah, we cannot do that though; the applicability of a composite rule must be the union of the applicabilities of the atomic rules it composes: https://www.w3.org/TR/act-rules-format/#applicability-composite

Reply:
But it will be. All atomic rules will have this applicability (I did not update them because I wanted to reach a stable definition of the applicability before moving onto the atomic rules).

Reply:
Gotcha, all good then! 😄